### DISCUSSION TITLE
Issue with the last update

### DISCUSSION THREAD

**Discussion by laraben** [livewire/livewire#3760] [7. Bugs (archived)] [answered]
Hi,

today i ran in some issue because of the update.

My app was working fine until i've updated livewire from 2.5 to 2.6

I couldn't find what was the issue for this so i've reverted the update and downgraded back to 2.5

Can someone tell me what could be the cause of this error?

<img width="725" alt="Schermafbeelding 2021-09-15 om 11 07 17" src="https://user-images.githubusercontent.com/34068668/133494772-f6fc69fa-0017-49a8-8702-43c3bedc7cfd.png">
<img width="766" alt="Schermafbeelding 2021-09-15 om 13 58 25" src="https://user-images.githubusercontent.com/34068668/133494735-6fdf5f9a-d104-4e8b-b673-74a8ad8a2686.png">

**Comment by joshhanley** [comment, marked as answer]
@laraben please see this comment for a solution https://github.com/livewire/livewire/discussions/3724#discussioncomment-1332569 🙂
